.buildkite/test-pipeline.yaml

```diff
@@ -1038,3 +1038,14 @@ steps:
   num_gpus: 2
   commands:
     - pytest -v -s tests/distributed/test_context_parallel.py
+
+##### RL Integration Tests #####
+- label: Prime-RL Integration Test # 5min
```

Comment:
we should put it in post-merge ci, hopefully we don't break the integration so often, so we don't need to test it so often.

Reply:
oh im not sure where that is. is there a different yaml for that? i see release and test, wasnt sure if there was another for merge

Reply:
Marking it optional is making it into post merge CI, in fact, it will only run nightly.